app/models/tagged_animal_assessment.rb

```diff
@@ -51,7 +51,8 @@ class TaggedAnimalAssessment < ApplicationRecord
     spawning_date
   ].freeze
 
-  validates :measurement_date, :shl_case_number, :spawning_date, :tag, :length, presence: true
+  validates :measurement_date, :spawning_date, presence: { message: "must be in the mm/dd/yy format" }
```

Comment:
would it be an issue if we can't determine b/w format errors vs real presence errors?

Reply:
I was trying to think about this and couldn't come up with any situations where it would matter. I initially thought we could do the presence check and a `validates :format` check, but since we're not saving the value in the `mm/dd/yy` format, it seemed better to me to just have the one check like above.